pages/api/mylist/[movieId].js

```diff
@@ -0,0 +1,36 @@
+import mongoClient from 'server/mongoDb';
+import withAuth from 'server/withAuth';
+async function handler(req, res) {
+  if (!['put', 'delete'].includes(req.method.toLowerCase())) {
+    return res.status(405).send();
+  }
```

Comment:
```suggestion
  const method = req.method.toLowerCase();
  if (!['put', 'delete'].includes(method)) {
    return res.status(405).send();
  }
```
and use `method` in the `if`s later on, for consistency.